src/generic_ui/polymer/settings.html

```diff
@@ -163,11 +160,10 @@
         </div>
 
         <div class='clickable currentAccountInfo' on-tap='{{ toggleAccountChooser }}'>
-          <div horizontal layout>
+          <div horizontal center layout>
             <span class='subdued' flex>{{ connectedNetworks }}</span>
-            <span hidden?='{{ model.onlineNetworks.length === 1 }}'>
```

Comment:
Is there any way to keep this in, I kind of like the logic flow of having the two hiddens separate.

Regardless, :thumbsup: